packages/front-end/components/Tabs/TabButton.tsx

```diff
@@ -57,6 +59,27 @@ export default function TabButton({
       ) : (
         ""
       )}
+      {notificationCount ? (
+        <div
+          className={`position-absolute badge d-flex justify-content-center align-self-center mr-1`}
+          style={{
+            zIndex: 1,
+            width: 18,
+            height: 18,
+            right: 0,
+            top: 3,
+            borderRadius: 50,
+            backgroundColor: "red",
+            color: "white",
+            lineHeight: 0.7,
+            boxShadow: "0 0 5px #000",
```

Comment:
This looks cleaner IMO and works better in dark mode
<img width="153" alt="image" src="https://github.com/growthbook/growthbook/assets/7927873/e13a371e-77e7-44d2-b86a-7b1fe6ce262c">

```suggestion
          style={{
            zIndex: 1,
            width: 20,
            height: 20,
            right: 0,
            top: 3,
            borderRadius: 50,
            backgroundColor: "#f00",
            color: "#fff",
            lineHeight: 0.7,
            boxShadow: "0 1px 2px #00000036",
            border: "1px solid #fff",
```